### DISCUSSION TITLE
Can we disable the build of all languages?

### DISCUSSION THREAD

**Discussion by ilzrv** [elreco/vue-tailwind-datepicker#43] [Q&A]
Can we disable the build of all languages if only 1 is used? (I use Vite)

<img width="422" alt="image" src="https://user-images.githubusercontent.com/28765966/209677762-f77a304f-03dc-4231-b8cf-59be35450c11.png">

**Comment by DirkBellstaedt** [comment]
I would also like to know if its possible,

**Comment by erick661** [comment]
I have the same problem

**Reply by elreco** [reply]
Available in the 1.7.0
